stories/Icons-stories.tsx

```diff
@@ -28,51 +30,90 @@ const meta = {
 
 export default meta;
 
-function renderIcons() {
+function renderIcons(regexp?: RegExp) {
   return Object.values(icons)
     .filter((icon) => !['IconWrapper'].includes(icon.name))
+    .filter((icon) => !regexp || regexp.exec(icon.name) !== null)
     .map((Icon) => {
       return (
```

Comment:
NP - this was from before, but we could use an arrow function here too and remove the `return`